shell/platform/darwin/ios/framework/Source/FlutterHeadlessDartRunner.mm

```diff
@@ -21,6 +19,7 @@
 #include "flutter/shell/platform/darwin/common/command_line.h"
 #include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h"
 #include "flutter/shell/platform/darwin/ios/framework/Source/FlutterDartProject_Internal.h"
+#include "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h"
```

Comment:
Several of these are ObjC, unless I'm misremembering. And they certainly have ObjC names.

Reply:
Yep good catch -- done.

Reply:
Actually still a couple more -- will get those momentarily.

Reply:
Done. And fixed a missing copyright header in one of the files.